src/ui/settings/ViewNetworkSettings.qml

```diff
@@ -98,6 +98,26 @@ Item {
                 }
             }
 
+            VPNCheckBoxRow {
+                id: multihopTunnel
+                objectName: "settingMultihopTunnel"
+                visible: VPNFeatureList.multihopSupported
+                Layout.rightMargin: Theme.windowMargin
+
+                //% "Multihop tunnel"
+                labelText: qsTrId("vpn.settings.multihop")
```

Comment:
These two strings just landed outside of strings.yaml (I thought the idea was to always use that from now on?).

More important, they conflict with the terminology used in https://github.com/mozilla-mobile/mozilla-vpn-client/pull/1573 (multi-hop vs multihop)

Which one is correct?

There seems to be also differences in terms of implementation, since I can't find anything similar to these strings in the PR.

Reply:
Ah, right, I forgot about the change to use strings.yaml. Sorry about that.

For what it's worth, this setting should only be temporary until such time as the UX for multihop is completed. So, hopefully we can simply remove these strings before code freeze.